docs/script.rst

```diff
@@ -714,80 +705,78 @@ Table of optional parameters:
 ==================  ================
 Method              Description
 ==================  ================
-hard                When ``true`` it creates an *hard* link, otherwise it creates a *soft* (aka *symbolic*) link. (default: ``false``).
-overwrite           When ``true`` it overwrite any exiting file with the same name, otherwise it will throws a `FileAlreadyExistsException <http://docs.oracle.com/javase/8/docs/api/java/nio/file/FileAlreadyExistsException.html>`_ (default: ``false``).
+hard                When ``true`` create a *hard* link, otherwise create a *soft* (aka *symbolic*) link. (default: ``false``)
+overwrite           When ``true`` overwrite any exiting file with the same name, otherwise throw a `FileAlreadyExistsException <http://docs.oracle.com/javase/8/docs/api/java/nio/file/FileAlreadyExistsException.html>`_ (default: ``false``)
 ==================  ================
```

Comment:
Why `create` here is not *third-person* ?

Reply:
I think third-person would be OK, too (_creates a hard link_, _overwrite any existing file_ (ha! another typo: current text has "exiting" -- I'll fix that)) My feeling was that these options are like _commands_ to the method, telling it to do something, thus the imperative tone. For example, look at `man ls` on your system and note the description of the switches (_*print* C-style escapes_, _*list* entries by columns_, etc.)

I left the descriptions of methods in third-person, as they read well, e.g. _getText... Returns the the file content as a string value_. A method _does_ something. But the options don't _do_ anything, they just tell the method how to do it, and the method _does_ it. That's my thinking, anyway :)

I'm happy to change this if you prefer second-person language. But then I suppose we should also change the table of `file`'s options, too, as they are the same.

Reply:
I see nice. Just for consistency with other table let's use the third person here as well. Then I will merge this PR. Thanks!